DOC-2068 - Management Appliance default drive selection behaviour (#7725)
spectrocloud/librarium: _partials/self-hosted/management-appliance/_installation-steps-prereqs.mdx

    - The second disk must be at least 500 GB and is used for the storage pool.

    :::tip

>     The largest drive is automatically chosen for the ISO stack. As such, it is recommended that the first disk has more storage than the second disk.

Reviewer: ```suggestion
    The largest drive is automatically selected for the ISO stack. Therefore, it is recommended that the first disk has more storage than the second disk.
```
